org.adempiere.ui.zk/WEB-INF/src/org/adempiere/webui/apps/LabelsSearchController.java

```diff
@@ -144,33 +144,37 @@ public void onSearchEcho(String value) {
 			boolean found = false;
 			int currentIndex = 0;
 			int foundIndex = 0;
-			
-			String sql = "SELECT"
-					+ "   AD_Label.AD_Label_ID,"
-					+ "   AD_Label.Name,"
-					+ "   COUNT(AD_LabelAssignment.*),"
-					+ "   CASE " 
-					+ "     WHEN AD_Label.AD_LabelCategory_ID > 0 AND AD_LabelCategory.IsCanBeUsedInAllTables = 'N'"
-					+ "       AND AD_LabelCategoryTable.AD_Table_ID <> ? THEN 'N'"
-					+ "     ELSE 'Y'"
-					+ "   END AS IsAllowed"
-					+ " FROM AD_Label"
-					+ " LEFT JOIN AD_LabelAssignment ON AD_LabelAssignment.AD_Label_ID = AD_Label.AD_Label_ID"
-					+ " LEFT JOIN AD_LabelCategory ON AD_LabelCategory.AD_LabelCategory_ID = AD_LabelCategory.AD_LabelCategory_ID"
-					+ " LEFT JOIN AD_LabelCategoryTable ON AD_LabelCategoryTable.AD_LabelCategory_ID = AD_LabelCategory.AD_LabelCategory_ID"
-					+ " WHERE AD_Label.AD_Client_ID = ? AND AD_Label.Name ILIKE ?"
-					+ " GROUP BY AD_Label.AD_Label_ID, AD_Label.Name, IsAllowed"
-					+ " ORDER BY COUNT(AD_LabelAssignment.*) DESC";
-					//+ " FETCH FIRST 15 ROWS ONLY";
-			
+
+			StringBuilder sql = new StringBuilder();
+			sql.append("SELECT ");
+			sql.append("   AD_Label.AD_Label_ID, ");
+			sql.append("   AD_Label.Name, ");
+			sql.append("   COUNT(*), ");
```

Comment:
I've not tested this but is "COUNT(AD_LabelAssignment.*)" and Count(*) the same since AD_LabelAssignment is outer join here ?